Source/WarpX.cpp

```diff
@@ -853,6 +853,10 @@ WarpX::ReadParameters ()
         pp_warpx.query("do_pml_j_damping", do_pml_j_damping);
         pp_warpx.query("do_pml_in_domain", do_pml_in_domain);
         pp_warpx.query("do_similar_dm_pml", do_similar_dm_pml);
+        v_particle_pml = PhysConst::c;
+        pp_warpx.query("v_particle_pml", v_particle_pml);
```

Comment:
Could you use the following code? This lets the user enter the velocity in units of c, which is more pratical.
```suggestion
        // Read `v_particle_pml` in units of the speed of light
        v_particle_pml = 1;
        pp_warpx.query("v_particle_pml", v_particle_pml);
        // Scale by the speed of light
        v_particle_pml = v_particle_pml * PhysConst::c;
```